### PR TITLE
v1.13 Changelog

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,33 @@
 # Mattermost Mobile Apps Changelog
 
+## v1.13.0 Release
+- Release Date: October 16, 2018
+- Server Versions Supported: Server v5.4+ is required, Self-Signed SSL Certificates are not supported
```
Comment (enahum) [inline CHANGELOG.md]: This is not true, the mobile app still supports server version 4.10+ which is the ESR but **IF** you have a 5.4 server the minimum app version is 1.13.

Example app version 1.12 and server 5.4 file upload won't work but app version 1.13 and server 4.10 works

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,33 @@
...
+
+### Highlights
+
+#### 
+
+### Improvements
+- Added support for message dropdowns.
+- Added support for hashtags.
+- Added support for nicknames on user profile.
+- Added ability to view emoji reactions.
```
Comment (enahum) [inline CHANGELOG.md]: to view who reacted to a post

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,33 @@
...
+- Added support for searching in direct and group message channels using the "in:" modifier.
```
Comment (enahum) [inline CHANGELOG.md]: only with server 5.4+

**Comment by amyblais** [comment]
@enahum FYI, this is still WIP.

**Comment by amyblais** [comment]
This is good for reviews now.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,45 @@
...
+#### Hashtags
+- Added support for hashtags.
+
+### Improvements
+- Added support for iPhone XR, XS and XS Max.
+- Added support for nicknames on user profile.
+- On servers 5.4+, added support for searching in direct and group message channels using the "in:" modifier.
+- Channel autocomplete now gets closed if multiple tildes are typed.
+- Added a draft icon in sidebar and channel switcher for channels with unsent messages.
+- Added paging for elasticsearch results.
```
Comment (enahum) [inline CHANGELOG.md]: Paging for search, the app does not discriminate, perhaps the server does bjt we don’t care?

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,45 @@
...
+
+### Bug Fixes
+- Refactored postlist to include Android Pie fixes and smoother scrolling.
+- Fixed an issue where deactivated users were not marked as such in "Jump To" search.
+- Fixed an issue where users got a permission error when trying to open a file from within the image preview screen.
+- Fixed an issue where session expiry notifications were not being sent on Android.
+- Fixed an issue where post attachments failed to upload.
+- Fixed an issue where the "DM More..." list cut off user info.
+- Fixed an issue where the user would see a system message flash before the thread loaded when opening a reply thread from, e.g. Recent Mentions list.
+- Fixed an issue on Android where the download button on images did not work.
```
Comment (enahum) [inline CHANGELOG.md]: This was working in 1.12 was a bug fix during this release cycle

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,45 @@
...
+- Fixed an issue where the error message was incorrectly formatted if the login method was set to email/password and the user tried to log in with SAML.
+- Fixed an issue on Android where the keyboard sometimes overlapped the bottom of the post textbox.
+- Fixed an issue where image thumbnails were not centered.
```
Comment (enahum) [inline CHANGELOG.md]: This was also not in 1.12 as the one above

**Comment by amyblais** [comment]
Comments addressed,

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,42 @@
 # Mattermost Mobile Apps Changelog
 
+## v1.13.0 Release
+- Release Date: October 16, 2018
+- Server Versions Supported: Server v4.10+ is required, Self-Signed SSL Certificates are not supported
+- Mobile Versions Supported: Mobile app version 1.13+ is required for servers 5.4+
```
Comment (esethna) [inline CHANGELOG.md]: "Mobile Versions Supported" doesn't really make sense

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,5 +1,43 @@
...
+- Added a draft icon in sidebar and channel switcher for channels with unsent messages.
+- Users are now redirected to the archived channel view (rather than to Town Square) when a channel is archived.
+- When closing an archived channel, users are now returned to the previously viewed channel.
+
+### Bug Fixes
+- Refactored postlist to include Android Pie fixes and smoother scrolling.
+- Fixed an issue where deactivated users were not marked as such in "Jump To" search.
+- Fixed an issue where users got a permission error when trying to open a file from within the image preview screen.
+- Fixed an issue where session expiry notifications were not being sent on Android.
+- Fixed an issue where post attachments failed to upload.
```
Comment (esethna) [inline CHANGELOG.md]: Should this say "- On servers 5.4+, fixed an issue where post attachments failed to upload."

**Comment by amyblais** [comment]
@enahum Can you help merge this when you have a chance?

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,5 +1,43 @@
...
+- Added a draft icon in sidebar and channel switcher for channels with unsent messages.
+- Users are now redirected to the archived channel view (rather than to Town Square) when a channel is archived.
+- When closing an archived channel, users are now returned to the previously viewed channel.
+
+### Bug Fixes
+- Refactored postlist to include Android Pie fixes and smoother scrolling.
+- Fixed an issue where deactivated users were not marked as such in "Jump To" search.
+- Fixed an issue where users got a permission error when trying to open a file from within the image preview screen.
+- Fixed an issue where session expiry notifications were not being sent on Android.
+- On servers 5.4+, fixed an issue where post attachments failed to upload.
```
Comment (esethna) [inline CHANGELOG.md]: @enahum can you help clarify this one. In what cases were post attachments broken? What did people experience using 1.12 with 5.3 or less?

Comment (enahum) [reply CHANGELOG.md]: The issue here is that a security fix went in for 5.4+ servers that breaks file uploads in mobile versions 1.12 or lower, the 1.13 version does handle the security fix that was introduced in 5.4.

* 5.4+ server app 1.12 or lower file upload fails
* 5.3 or lower server app version 1.12 or lower file uploads works
* any server version and app version 1.13+ file upload works

**Comment by amyblais** [comment]
Can this PR be merged and then open a new one for any needed tweaks? v1.13 has been released so people may be looking for this and it also includes the important compatibility note.

Again, I don't have merge permissions.

**Pull request merged**